bin/get_workspace_status

```diff
@@ -72,3 +72,4 @@ echo buildHost          "$(hostname -f)"
 # used by mixer/broker version package
 echo "buildVersion ${VERSION}"
 echo "buildStatus ${tree_status}"
+echo "buildID $(date -u +%Y-%m-%d-$(git rev-parse --short HEAD))"
```

Comment:
We shouldn't be going in the direction of adding in timestamps.  At most perhaps the time of the commit.
https://reproducible-builds.org/docs/timestamps/